eeps/eep-0045.md

```diff
@@ -1,8 +1,8 @@
     Author: Björn Gustavsson <bjorn(at)erlang(dot)org>
-    Status: Accepted/19.0  Implemented in OTP version 19.0
+    Status: Accepted/19.0  Implemented in OTP release 19
```

Comment:
Here is the mention of 19.0 and release 19.
When is ok to say 19.0 and when is ok to simply say release 19?
Maybe this is something that should be in the description, so that we make some guidelines?

Reply:
19.0 is a version and 19 is a release. So depending on what you talk about to use either or.